dashboard/app/views/levels/_project_ui.html.haml

```diff
@@ -23,6 +24,28 @@
     $('#save-project').show();
   }
 
+  var createApp = function(options) {
+    var defaultOptions = {
+      name: "Untitled",
+      createdAt: new Date,
+      updatedAt: new Date,
+      levelId: appOptions.levelId, // TODO this is not actually a unique identifier for the level
```

Comment:
yep... the intention was to use this for the standalone project urls, since we only need to know the level, but now we actually need to know the script as well (for these project levels).

(currently we use the levelUrl which happens to handle either case, but that's kind of a hack).